src/scripts/main.js

```diff
@@ -6,3 +6,41 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+
+people.forEach(person => {
+  const table = document.querySelector('.dashboard');
```

Comment:
Instead of selecting the dashboard table element within each iteration of the loop, you can move the selection outside the loop to improve performance.